js/ui/tabs.js

```diff
@@ -265,8 +267,8 @@ var Tabs = CollectionWidget.inherit({
     },
 
     _renderScrolling: function() {
-        this.$element().removeClass(TABS_EXPANDED_CLASS);
-        this.$element().removeClass(OVERFLOW_HIDDEN_CLASS);
+        let removeClasses = [TABS_STRETCHED_CLASS, TABS_EXPANDED_CLASS, OVERFLOW_HIDDEN_CLASS];
```

Comment:
If you want to use the ES6 syntaxis then you need to replace all "require" to "import from" and use "const" instead of "let". Or use "var" of ES5.